kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/Coordinator.java

```diff
@@ -140,7 +137,7 @@ private void doCommit(boolean partialCommit) {
     Map<TableIdentifier, List<Envelope>> commitMap = commitState.tableCommitMap();
```

Comment:
@fqaiser94 awkward here.  Bryan has moved this all over to `TableReference` which you can no longer use as a key (as it contains a schema reference, which tends to make it non-unique).  Except it means we get out of TableReference into TableIdentifier just to make a TableReference again later...